randobot/handler.py

```diff
@@ -161,13 +173,512 @@ async def race_data(self, data):
             await self.unpin_message(self.state['pinned_msg'])
             del self.state['pinned_msg']
 
+    @monitor_cmd
+    async def ex_s7(self, args, message):
+        """
+        Handle !s7 commands.
+
+        Set up room for Draft Mode.
+        """
+        if self._race_in_progress() or not self._is_s7_race():
+            return
+        
+        draft = self.state.get('draft_data')
+
+        # Handle valid arguments.
+        if len(args) >= 1 and args[0] in ('tournament', 'practice', 'qualifier', 'cancel'):
```

Comment:
Think there shuld be a message if `len(args) == 0` (wasn't there one before? or am I imagining that)

Reply:
There was not but I'll add one indicating that you must supply arguments